angular-chart.js

```diff
@@ -362,5 +365,11 @@
       var options = angular.extend({}, Chart.defaults.global, ChartJs.getOptions(type), scope.options);
       return options.responsive;
     }
+
+    function destroyChart(chart, scope) {
+      if(!chart) return;
+      chart.destroy();
+      scope.$emit('destroy', chart);
```

Comment:
I thought about this too. I was trying to be consistent with the other events and it may be useful for debugging. I don't have a strong opinion either way - let me know and I'll keep it or remove it.